packages/labs/compiler/test_files/raw_text_element_edgecase.js

```diff
@@ -0,0 +1,13 @@
+// Ensure that the raw text content in the `<style>` tag does not get
+// miscompiled into `<?>`.
+import { html, render } from 'lit';
+
+render(html`<style>
+  div::before {
+    content: '<!---->';
+  }
+  div::after {
+    content: '<!--?-->';
+  }
+</style>
+<div>Hello world</div>`, document.body);
```

Comment:
Nit: newline at end of file
```suggestion
<div>Hello world</div>`, document.body);

```